src/test/regress/multi_mx_schedule

```diff
@@ -25,6 +25,7 @@ test: multi_mx_create_table
 test: start_stop_metadata_sync
 test: multi_mx_hide_shard_names
 test: multi_mx_add_coordinator
+test: multi_mx_connectivity
```

Comment:
would have been great to show that we can access to the coordinator if the coordinator is in the metadata.
Would you mind inlining the test file `multi_mx_connectivity` into `multi_mx_add_coordinator` so that we can show it?